### PR TITLE
Added version and build info functions

### PR DESCRIPTION
**Pull request by akallabeth**
Added functions to get
 * Version String
 * Build Time String
 * Git Revision String
and appropriate tests.

### CODE REVIEW

File: include/freerdp/freerdp.h [hunk 1]
```diff
@@ -260,6 +260,9 @@ FREERDP_API UINT32 freerdp_error_info(freerdp* instance);
 FREERDP_API void freerdp_set_error_info(rdpRdp* rdp, UINT32 error);
 
 FREERDP_API void freerdp_get_version(int* major, int* minor, int* revision);
+FREERDP_API char* freerdp_get_version_string(void);
+FREERDP_API char* freerdp_get_build_date(void);
+FREERDP_API char* freerdp_get_build_revision(void);
```
Comment (mfleisz) [inline include/freerdp/freerdp.h]: Shouldn't these functions return const char* ?

File: winpr/include/winpr/winpr.h [hunk 2]
```diff
@@ -47,4 +47,9 @@
 #define INLINE	inline
 #endif
 
+WINPR_API void winpr_get_version(int* major, int* minor, int* revision);
+WINPR_API char* winpr_get_version_string(void);
+WINPR_API char* winpr_get_build_date(void);
+WINPR_API char* winpr_get_build_revision(void);
+
```
Comment (mfleisz) [inline winpr/include/winpr/winpr.h]: Shouldn't these functions return const char* ?

File: include/freerdp/freerdp.h [hunk 3]
```diff
@@ -260,6 +260,9 @@ FREERDP_API UINT32 freerdp_error_info(freerdp* instance);
...
```
Comment (mfleisz) [inline include/freerdp/freerdp.h]: Shouldn't these functions return const char* ?

**Comment by mfleisz** [comment]
+1

**Comment by hardening** [comment]
+1

**Pull request merged**